src/02_spec/07_modules/cdm/dbgregisters.rst

```diff
@@ -0,0 +1,229 @@
+
+  * - 0x0290
+    - ``DEBUG_VALUE_REG_5_1``
+    - Bits [31:16] of DVR5
+
+  * - 0x0291
+    - ``DEBUG_VALUE_REG_6_0``
+    - Bits [15:0] (least significant bits) of DVR6
+
+  * - 0x0292
+    - ``DEBUG_VALUE_REG_6_1``
+    - Bits [31:16] of DVR6
+
+  * - 0x0293
+    - ``DEBUG_VALUE_REG_7_0``
+    - Bits [15:0] (least significant bits) of DVR7
+
+  * - 0x0294
+    - ``DEBUG_VALUE_REG_7_1``
+    - Bits [31:16] of DVR7
+
+  * - 0x0295
+    - ``DEBUG_CTRL_REG_0_0``
+    - Bits [15:0] (least significant bits) of DCR0
+
+  * - 0x0296
+    - ``DEBUG_CTRL_REG_0_1``
+    - Bits [31:16] of DCR0
+
+  * - 0x0297
+    - ``DEBUG_CTRL_REG_1_0``
+    - Bits [15:0] (least significant bits) of DCR1
+
+  * - 0x02A0
+    - ``DEBUG_CTRL_REG_1_1``
+    - Bits [31:16] of DCR1
+
+  * - 0x02A1
+    - ``DEBUG_CTRL_REG_2_0``
+    - Bits [15:0] (least significant bits) of DCR2
+
+  * - 0x02A2
+    - ``DEBUG_CTRL_REG_2_1``
+    - Bits [31:16] of DCR2
+
+  * - 0x02A3
+    - ``DEBUG_CTRL_REG_3_0``
+    - Bits [15:0] (least significant bits) of DCR3
+
+  * - 0x02A4
+    - ``DEBUG_CTRL_REG_3_1``
+    - Bits [31:16] of DCR3
+
+  * - 0x02A5
+    - ``DEBUG_CTRL_REG_4_0``
+    - Bits [15:0] (least significant bits) of DCR4
+
+  * - 0x02A6
+    - ``DEBUG_CTRL_REG_4_1``
+    - Bits [31:16] of DCR4
+
+  * - 0x02A7
+    - ``DEBUG_CTRL_REG_5_0``
+    - Bits [15:0] (least significant bits) of DCR5
+
+  * - 0x02B0
+    - ``DEBUG_CTRL_REG_5_1``
+    - Bits [31:16] of DCR5
+
+  * - 0x02B1
+    - ``DEBUG_CTRL_REG_6_0``
+    - Bits [15:0] (least significant bits) of DCR6
+
+  * - 0x02B2
+    - ``DEBUG_CTRL_REG_6_1``
+    - Bits [31:16] of DCR6
+
+  * - 0x02B3
+    - ``DEBUG_CTRL_REG_7_0``
+    - Bits [15:0] (least significant bits) of DCR7
+
+  * - 0x02B4
+    - ``DEBUG_CTRL_REG_7_1``
+    - Bits [31:16] of DCR7
+
+  * - 0x02B5
+    - ``DEBUG_MODE_REG_1_0``
+    - Bits [15:0] (least significant bits) of DMR1
+
+  * - 0x02B6
+    - ``DEBUG_MODE_REG_1_1``
+    - Bits [31:16] of DMR1
+
+  * - 0x02B7
+    - ``DEBUG_MODE_REG_2_0``
+    - Bits [15:0] (least significant bits) of DMR2
+
+  * - 0x02C0
+    - ``DEBUG_MODE_REG_2_1``
+    - Bits [31:16] of DMR2
+
+  * - 0x02C1
+    - ``DEBUG_WC_REG_0_0``
+    - Bits [15:0] (least significant bits) of Debug Watchpoint Counter 0
+
+  * - 0x02C2
+    - ``DEBUG_WC_REG_0_1``
+    - Bits [31:16] of Debug Watchpoint Counter 0
+
+  * - 0x02C3
+    - ``DEBUG_WC_REG_1_0``
+    - Bits [15:0] (least significant bits) of Debug Watchpoint Counter 1
+
+  * - 0x02C4
+    - ``DEBUG_WC_REG_1_1``
+    - Bits [31:16] of Debug Watchpoint Counter 1
+
+  * - 0x02C5
+    - ``DEBUG_STOP_REG_0``
+    - Bits [15:0] (least significant bits) of Debug Stop Register
+
+  * - 0x02C6
+    - ``DEBUG_STOP_REG_1``
+    - Bits [31:16] of Debug Stop Register
+
+  * - 0x02C7
+    - ``DEBUG_REASON_REG_0``
+    - Bits [15:0] (least significant bits) of Debug Reason Register
+
+  * - 0x02D0
+    - ``DEBUG_REASON_REG_1``
+    - Bits [31:16] of Debug Reason Register
+
```

Comment:
What's missing is the description what these registers do. See http://opensocdebug.readthedocs.io/en/latest/02_spec/07_modules/mam/dbgregisters.html (at the bottom) for an example. You can of course refer to the appropriate parts in the specification, but still a quick explanation here is helpful. Also describe there if the registers are read-only, write-only or read-write.